### PR TITLE
Allow concurrent use of el.cheapo

### PR DESCRIPTION
**Pull request by graetzer**
http://jenkins.arangodb.biz:8080/job/arangodb-matrix-pr/10423/

### CODE REVIEW

**Comment by graetzer** [comment]
http://jenkins01.arangodb.biz:8080/job/arangodb-matrix-pr/10528/

**Comment by graetzer** [comment]
http://jenkins01.arangodb.biz:8080/job/arangodb-matrix-pr/10530/

**Comment by graetzer** [comment]
http://jenkins01.arangodb.biz:8080/job/arangodb-matrix-pr/10589/

**Comment by graetzer** [comment]
http://jenkins01.arangodb.biz:8080/job/arangodb-matrix-pr/10671/

**Comment by graetzer** [comment]
http://jenkins01:8080/view/Java/job/arangodb-ANY-java-driver-matrix/935/

**Comment by graetzer** [comment]
http://jenkins01.arangodb.biz:8080/job/arangodb-matrix-pr/10682/

**Comment by graetzer** [comment]
http://172.16.10.101:8080/view/Java/job/arangodb-ANY-java-driver-matrix/940/

File: arangod/Transaction/ClusterUtils.cpp [hunk 1]
```diff
@@ -35,7 +35,23 @@ namespace arangodb {
 namespace transaction {
 namespace cluster {
 
+void abortTransactions(LogicalCollection& coll) {
+  transaction::Manager* mgr = transaction::ManagerFeature::manager();
+  if (!mgr) { // might be called during shutdown
```
Comment (mpoeter) [inline arangod/Transaction/ClusterUtils.cpp:40]: This looks very much like a race condition. Suppose this is called during shutdown, but _before_ `ManagerFeature::unprepare`. Then we would get a valid `ptr` to an object that can be destroyed at any time.

Comment (graetzer) [reply arangod/Transaction/ClusterUtils.cpp:40]: not gonna happen, comment is wrong

Comment (graetzer) [reply arangod/Transaction/ClusterUtils.cpp:40]: only nullptr during unit testing

**Comment by fceller** [comment]
http://jenkins.arangodb.biz:8080/view/PR/job/arangodb-matrix-pr-full/1/

**Pull request merged**